.eslintrc.json

```diff
@@ -42,11 +42,11 @@
         "props": false
       }
     ],
-    "complexity": [ 2, 3 ],
-    "max-depth": [2, 3],
-    "max-len": [1, 160, 4],
+    "complexity": [ 2, 10 ],
```

Comment:
Is there a way that we can keep the previous values and make the code work in that way in order to keep things more simple